CMakeLists.txt

```diff
@@ -0,0 +1,353 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(libcoap VERSION 4.3.0 LANGUAGES CXX C)
+
+set(LIBCOAP_API_VERSION 2)
```

Comment:
Likewise `LIBCOAP_API_VERSION 2` could then be `LIBCOAP_API_VERSION @LIBCOAP_API_VERSION@`

Reply:
I think the effort to make this importable is not worth as this changes very infrequently, is only here for doxygen.in and can be changed if needed easily